dcompiler.dd

```diff
@@ -764,6 +781,8 @@ $(H2 $(LNAME2 executable, Building Executables))
     invocation of $(B dmd) is that $(B dmd) will be able to do some level
     of cross-module optimizations, such as function inlining across modules.
     )
+    $(P The $(D -i) flag can be used to let $(B dmd) automatically search and compile
+    all depending source files.)
```

Comment:
The text above talks about:

>  dmd can build an executable much faster if as many of the source files as possible are put on the command line.

Well, nowadays that's no longer necessary with `-i` (though I wasn't sure on the wording or if we should modify the existing paragraph.

Reply:
I suggest the following wording instead:

> $(P The $(D -i) flag can be used to automatically compile imported modules)